libsolidity/formal/SMTEncoder.h

```diff
@@ -440,6 +441,7 @@ class SMTEncoder: public ASTConstVisitor
 	bool m_checked = true;
 
 	langutil::UniqueErrorReporter& m_errorReporter;
+	langutil::UniqueErrorReporter& m_unsupportedErrorAcc;
```

Comment:
What does the `Acc` stand for? Accumulated? I think the var name could be a little bit clearer, and if not possible, a comment at least would have been nice.